notebooks/GWAS_blood_pressure_p3/main.py

```diff
@@ -0,0 +1,647 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Title: 3-GWAS-secondary-analysis
+
+# Notebook author: Beth Sheets
+# Herzogification: Ash O'Farrell
+
+import os
+from unittest import mock
+
+import herzog
+
+
+# Heavyweight dependencies (e.g. hail, Spark) make it challenging to create robust automated testing for this notebook.
+# In order to get it to pass Python execution, many objects are mocked with unittest.mock.MagicMock. These have been
+# marked with `# test fixture`. This will at least catch some syntax errors.
+#
+# In the meantime, this notebook should be tested manually in a Terra notebook environment.
+
+consolidate_gen3_pheno_tables = mock.MagicMock()  # test fixture
+get_terra_table_to_df = mock.MagicMock()  # test fixture
+get_ipython = mock.MagicMock()  # test fixture
+os.environ['GOOGLE_PROJECT'] = "foo"  # test fixture
+os.environ['WORKSPACE_BUCKET'] = "bar"  # test fixture
+
+with herzog.Cell("markdown"):
+    """
+    # Data disclaimer
+
+    All data in this notebook is restricted access. You are responsible for the protection and proper use of any data or files downloaded from this workspace.
+
+    # Introduction
+
+    This notebook demonstrates typical initial steps in a genetic association analysis: exploring phenotype distributions, filtering and LD-pruning, and Principal Component analysis.
+
+    This notebook hopes to help you understand the following steps in performing an association test in BioData Catalyst:
+
+    1. First, we discover all of the metadata available from our Gen3 export. We then use a set of functions created to merge and reformat the metadata to create a consolidated data table. We reformat a bit of the Gen3 graph language to be more familiar TOPMed nomenclature.
```

Comment:
Remove lines 35-47. Instead I suggest:
"This genotypic analysis notebook leads you through the steps to prepare you genomic data for association testing. 
1. You will import the phenotype.csv you generated in "TITLE_OF_PRIOR_NOTEBOOK" notebook. 
2. Next, we import, explore, and perform quality control on genotypic data. 
3. and so on...